user.go

```diff
@@ -190,6 +190,19 @@ func (c *Client) FlagUser(ctx context.Context, targetID, flaggedBy string) (*Res
 	return &resp, err
 }
 
+type GuestUserResponse struct {
+	User        *User  `json:"user"`
+	AccessToken string `json:"access_token"`
+	Response
+}
+
+// SetGuestUser creates a new guest user.
+func (c *Client) SetGuestUser(ctx context.Context, user *User) (*GuestUserResponse, error) {
```

Comment:
Not keen with that naming. What about `CreateGuestUser` or `GetGuestUser` ? feel free to ignore me :)

Reply:
✅